conf/test_full.config

```diff
@@ -14,23 +14,19 @@ params {
     config_profile_name        = 'Full test profile'
     config_profile_description = 'Full test dataset to check pipeline function'
 
-    // Input data for full size test
-    // TODO nf-core: Specify the paths to your full test data ( on nf-core/test-datasets or directly in repositories, e.g. SRA)
-    // TODO nf-core: Give any required params for the test so that command line flags are not needed
 
     // Genome references
-    map             = "/groups/dog/llenezet/test-datasets/data/genetic_maps.b38/chr21.b38.gmap.gz"
+    map             = "https://bochet.gcc.biostat.washington.edu/beagle/genetic_maps/plink.GRCh38.map.zip"
     genome          = "GRCh38"
-    fasta           = "/groups/dog/llenezet/script/phaseimpute/data/genome.fa"
 
     // Resources increase incompatible with Github Action
     max_cpus   = 12
     max_memory = '50.GB'
     max_time   = '6.h'
 
     // Input data
-    input                  = "tests/csv/sample_sim.csv"
-    panel                  = "tests/csv/panel.csv"
+    input                  = "${projectDir}/tests/csv/sample_sim_full.csv"
+    panel                  = "${projectDir}/tests/csv/panel_full.csv"
     input_region_string    = "all"
     step                   = "simulate"
```

Comment:
The aim would effectively be to perform simulation, pre-processing, imputation and validation.
I did not have yet the time to work on the simulation tests.
This will be done in another PR